src/effects/backends/builtin/pitchshifteffect.cpp

```diff
@@ -134,19 +136,30 @@ void PitchShiftEffect::processChannel(
             m_currentFormant != formantPreserving) {
         m_currentFormant = formantPreserving;
 
+        // Set the RubberBand option for the formant preserving.
         pState->m_pRubberBand->setFormantOption(m_currentFormant
                         ? RubberBand::RubberBandStretcher::
                                   OptionFormantPreserved
                         : RubberBand::RubberBandStretcher::
                                   OptionFormantShifted);
     }
 
+    // Get the parameter value of the Pitch knob
+    // based on the current setting of the Range knob.
```

Comment:
This comment does also not add any value. 
You may explain that one of from 0...X and the other from -x to +y ... and describe the implications.